resources/views/components/logo.blade.php

```diff
@@ -1 +1,11 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 300 300"><path fill="#2EBA86" d="M93.6 61.95h19.05c7.05 0 12.75 5.7 12.75 12.75s-5.7 12.75-12.75 12.75H93.6c-7.05 0-12.75-5.7-12.75-12.75S86.7 61.95 93.6 61.95ZM46.05 162.6h60.3c7.05 0 12.75 5.7 12.75 12.75s-5.7 12.75-12.75 12.75h-60.3c-7.05 0-12.75-5.7-12.75-12.75 0-7.35 5.7-12.75 12.75-12.75Z"/><path fill="#203F60" fill-opacity=".95" d="M244.5 61.95h-93.75c-7.05 0-12.75 5.7-12.75 12.75 0 6 4.5 11.1 10.2 12.45h20.85c5.7 1.2 10.2 6.3 10.2 12.45 0 7.05-5.7 12.75-12.75 12.75H84.75c-5.7 1.2-10.2 6.3-10.2 12.45 0 6.15 4.5 11.1 10.2 12.45h88.35c7.05 0 12.75 5.7 12.75 12.75s-5.7 12.75-12.75 12.75h-31.2c-5.7 1.2-10.2 6.3-10.2 12.45 0 6.15 4.5 11.1 10.2 12.45h12c.9 0 1.65 0 2.55.3.3 0 .3 0 .6.3.6 0 .9.3 1.65.3.3 0 .6.3.9.3.3.3.9.3 1.2.6.3.3.6.3.9.6l.9.9.6.6.9.9c.3.3.3.6.6.9.3.3.3.9.6 1.2 0 .3.3.6.3.9.3.6.3.9.3 1.65 0 .3 0 .6.3.6 0 .6.3 1.65.3 2.55 0 7.05-5.7 12.75-12.75 12.75H33.3c-.9 0-1.65 0-2.55-.3h-16.5c-7.05 0-12.75 5.7-12.75 12.75s5.7 12.75 12.75 12.75H184.2l11.7-66.45h10.2l24.75 66.45h48.3l-28.2-70.2c28.65-7.95 44.55-28.95 48.3-50.85 5.7-30.45-11.55-55.2-54.75-55.2Zm10.8 57.15c-2.55 13.65-11.4 22.2-27.6 22.2H201l7.65-44.1h26.7c16.5 0 22.5 8.55 19.95 21.9Z"/></svg>
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_10342_2)">
```

Comment:
We'd some issues with ID's here when the logo is used 2+ times on a page. @JimmyHoenderdaal knows more on that.